### ISSUE TITLE
Specular Glossiness workflow

### ISSUE THREAD

**Issue by MGDegenhardt** [armory3d/armortools#737]
Actually Armor Paint uses the PBR Metallic Roughness workflow, and it is great, but there's a lot of engines that uses the PBR Specular Glossiness workflow to texture the models, so I'm wondering if there's a way to add a new Material Output node with this feature, or maybe, a dropdown menu in the node that allow user to select the desired workflow.
I have made a sketch for what I'm proposing:
![Two_Nodes](https://user-images.githubusercontent.com/69319609/103292016-ba6fa600-49cb-11eb-8aa0-d6c9d23b2706.png)
![One_only_node](https://user-images.githubusercontent.com/69319609/103292030-c5c2d180-49cb-11eb-9ddc-c90af3dbb446.png)

**Comment by MathemanFlo** [comment]
Maybe an intermediate solution could be to let everything as it is and to enhance the exporter to support Specular-Glossiness-Diffuse PBR workflow engines. Glossiness is the same as smoothness, which is already supported in the exporter (it is just the inverse of roughness). The specular and diffuse textures could be computed from basecolor and metallic because every base color, metallic tuple can be transformed to a specular diffuse tuple.

**Comment by MGDegenhardt** [comment]
I agree with "Glossiness is the same as smoothness", but I guess this is
for the mesh itself, but what happens when you import a Glossy from a
texture?
What I'm doing is to use an invert node between the image node and the
Material output, so my glossy map changes to the rough map and solves the
problem, but, in the next textures I will do the same over and over again.
It is tedious!
This is why I think a specific node coil help.


Em ter., 29 de dez. de 2020 às 13:29, MathemanFlo <notifications@github.com>
escreveu:

**Comment by luboslenco** [comment]
Would love to do this, but still have too many other issues to handle, sorry. I am adding a backlog tag and hope to revisit it eventually.
